### PR TITLE
[dwdunwetter] Migrate to bnd build

### PR DESCRIPTION
**Pull request by wborn**
Migrates the DwdUnwetter Binding to the bnd build.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -224,6 +222,7 @@
 /bundles/org.openhab.binding.bigassfan/ @mhilbush
 /bundles/org.openhab.binding.dmx/ @J-N-K
 /bundles/org.openhab.binding.freebox/ @lolodomo
+/bundles/org.openhab.binding.dwdunwetter/ @limdul79
 /bundles/org.openhab.binding.globalcache/ @mhilbush
```
Comment (J-N-K) [inline CODEOWNERS]: Wrong sorting. dmx / dwdunwetter / freebox

File: bundles/pom.xml [hunk 2]
```diff
@@ -25,6 +25,7 @@
     <module>org.openhab.binding.bigassfan</module>
     <module>org.openhab.binding.dmx</module>
     <module>org.openhab.binding.freebox</module>
+    <module>org.openhab.binding.dwdunwetter</module>
     <module>org.openhab.binding.globalcache</module>
```
Comment (J-N-K) [inline bundles/pom.xml]: Wrong sorting.

**Comment by wborn** [comment]
I don't know how that happend but it is fixed!

**Comment by davidgraeff** [comment]
@openhab/2-x-add-ons-maintainers : For the duration of the migration I suggest to have more people with push access in the maintainers group. I suggest @J-N-K and myself.

**Comment by martinvw** [comment]
@wborn as a first step I would say its okay to merge your own (infra) PR's with two approvals of experienced people such as @J-N-K / @davidgraeff .

@kaikreuzer I leave further decisions regarding access to you.

**Comment by kaikreuzer** [comment]
@davidgraeff Sorry, I didn't notice you weren't yet a member of @openhab/2-x-add-ons-maintainers, since you were supposed to be one (we had added you as an add-on maintainer a while ago, but this seems to have gone lost in this team). I have just added you now!

Wrt @J-N-K, @cweitkamp started a formal voting process, which should finish latest within a week.

**Pull request merged**